CargoMonitor/CargoMonitor.cs

```diff
@@ -27,6 +27,7 @@ public class CargoMonitor : EDDIMonitor
         public int cargocarried;
         private static readonly object inventoryLock = new object();
         public event EventHandler InventoryUpdatedEvent;
+        public bool testMode = false; // Set to true during testing to prevent writing to files
```

Comment:
I'm trying to avoid a scenario like we see in #77 where we write to permanent storage. Recommendations for changing this?

Reply:
Core recommendation is to factor logic methods out of the writing methods, and test the logic methods. 

From past experience this will produce a much cleaner architecture than a "test mode" bool.